Gems/Atom/RPI/Code/Include/Atom/RPI.Public/RPISystemInterface.h

```diff
@@ -64,7 +64,7 @@ namespace AZ
             
             //! Tick for graphics simulation that runs on the CPU. 
             //! This will drive FeatureProcessor simulation activity. It should be called once per game-tick.
-            virtual void SimulationTick() = 0;
+            virtual void SimulationTick(float deltaTime, float simulationTimeMs) = 0;
```

Comment:
suggestion: Could we add an API comment to explain exactly what `simulationTimeMs` (and `deltaTime`) are. Also should we remove `Ms` from the second parameter or add units to `deltaTime` as well? It just feels strange to have one and not the other.

Reply:
I reverted the API change to denoise this PR - it was mostly a convenience that I could pass things in from the tick bus, and it's not really that important.